packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -56,57 +56,66 @@
 	}
 }
 
-.block-editor-block-contextual-toolbar.is-fixed {
+.block-editor-block-contextual-toolbar {
 	position: sticky;
 	top: 0;
 	z-index: z-index(".block-editor-block-popover");
 	display: block;
 	width: 100%;
-}
+	// Block UI appearance.
+	background-color: $white;
+	flex-shrink: 3;
+
+	// Raise the specificity.
+	&.components-accessible-toolbar {
+		border: none;
+		border-bottom: $border-width solid $gray-200;
+		border-radius: 0;
+	}
 
-// on desktop browsers the fixed toolbar has tweaked borders
-@include break-medium() {
-	.block-editor-block-contextual-toolbar.is-fixed {
-		.block-editor-block-toolbar {
-			.components-toolbar-group,
-			.components-toolbar {
-				border-right: none;
-
-				&::after {
-					content: "";
-					width: $border-width;
-					margin-top: $grid-unit + $grid-unit-05;
-					margin-bottom: $grid-unit + $grid-unit-05;
-					background-color: $gray-300;
-					margin-left: $grid-unit;
-				}
-
-				& .components-toolbar-group.components-toolbar-group {
-					&::after {
-						display: none;
-					}
-				}
-			}
+	.block-editor-block-toolbar {
+		overflow: auto;
+		overflow-y: hidden;
 
-			> :last-child,
-			> :last-child .components-toolbar-group,
-			> :last-child .components-toolbar {
-				&::after {
-					display: none;
-				}
+		> :last-child,
+		> :last-child .components-toolbar-group,
+		> :last-child .components-toolbar {
+			&::after {
+				display: none;
 			}
 		}
 	}
-}
 
-.block-editor-block-contextual-toolbar.has-parent:not(.is-fixed) {
-	margin-left: calc(#{$grid-unit-60} + #{$grid-unit-10});
+	.block-editor-block-toolbar .components-toolbar-group,
+	.block-editor-block-toolbar .components-toolbar {
+		border-right-color: $gray-200;
+	}
 
-	.show-icon-labels & {
-		margin-left: 0;
+	& > .block-editor-block-toolbar {
+		flex-grow: initial;
+		width: initial;
+	}
+
+	.block-editor-block-parent-selector {
+		position: relative;
+
+		// Parent selector dot divider
+		&::after {
+			content: "\00B7";
```

Comment:
So are we keeping this dot divider instead of going with the default grey vertical lines? Why?

Reply:
@ellatrix the mid-dot was definitely an intentional design choice, to be minimalist, and because we have quite a few chevrons in the interface already. Notably there's also the expand/collapse double chevrons to collapse the toolbar in smaller viewports:

<img width="1472" alt="Responsive scheme" src="https://github.com/WordPress/gutenberg/assets/1204802/c6bdd7a2-0848-48aa-9980-2cfeafff3c53">

Speaking of, that seems broken in trunk at the moment. @jeryj @draganescu do you know what might have happened here?
![Screenshot 2023-12-20 at 10 32 38](https://github.com/WordPress/gutenberg/assets/1204802/d569cce3-7330-4c7a-9b0e-c3e2f9b28071)

Revisiting the mid-dot, here are some thoughts. What it looks like, more or less, at the moment:
<img width="848" alt="Current" src="https://github.com/WordPress/gutenberg/assets/1204802/e96a7b56-c2cb-4093-bb8d-81bffb4cb139">

Here's what it might look like if we added a couple of more dividers, one between the parent/child combo, and one _before_ the toolbar itself:

<img width="848" alt="divider" src="https://github.com/WordPress/gutenberg/assets/1204802/9b556e01-fd7a-41ed-8fb4-c904624c98c9">

I think this one might work well. But for what it's worth, here's what that same one would look like, with a chevron:

<img width="848" alt="chevron" src="https://github.com/WordPress/gutenberg/assets/1204802/aca6b157-3af7-47c0-aa8b-89a49b88dcce">

Reply:
> Speaking of, that seems broken in trunk at the moment. @jeryj @draganescu do you know what might have happened here?

We removed the collapse chevron from headers where there was no central command bar, as collapsing the toolbar would not reveal anything new.

Reply:
Ah right, thanks for the clarification. The reason we added the chevron in the first place was not just about the document toolbar element, but in fact about plugins that inject buttons into the DOM, like this:
![2023-12-21 09 50 48 local-wordpress local cd76c9fdef5a](https://github.com/WordPress/gutenberg/assets/1204802/0c401639-937d-4088-8908-91fa5c6f5669)

However I can see that since the block toolbar itself is no longer fixed position, it no longer covers those buttons, it just gets pushed rightwards — arguably a better way to fix it. 

I would still restore this chevron, however. Ultimately the editors are meant to be unified, not different, there are a lot of PRs underway to make this happen, so it's a single experience, a single codebase. So I'd restore it, we'll likely see the document title element come to the post and page editors eventually. It'll also fix this issue where there's a trailing separator that was added only to separate the toolbar itself from the now-missing chevron:

![Screenshot 2023-12-21 at 09 53 10](https://github.com/WordPress/gutenberg/assets/1204802/4e98d0a6-1808-4979-b839-9d66f2c450b7)